scripts/darwin/build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+# Load system env only
+source /etc/bashrc
+source /etc/profile
+
+# Make sure home is pointing to current directory
+export HOME=$(pwd)
+# Calculate number of available cores
+export J=$(( $(nproc --all) )) && echo Using ${J} cores during build
+
+# Load system modules
+module purge
+module load cmake/3.11.1
+module load gcc/7.4.0
+module load clang/8.0.0
```

Comment:
Could you add `export NVCC_WRAPPER_DEFAULT_COMPILER=g++` somewhere before the cmake configure so that it's explicit we're using GCC? I think nvcc_wrapper defaults to a GCC compiler, so I don't think it's broken right now, but seeing clang loaded _after_ gcc is confusing at first glance

Reply:
This has been addressed.